sdk/cognitivelanguage/ai-language-conversations/src/conversationAnalysisClient.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+import { bearerTokenAuthenticationPolicy } from "@azure/core-rest-pipeline";
+import { PollerLike, PollOperationState } from "@azure/core-lro";
+import {
+  ConversationAnalysisClientOptionalParams,
+  AnalyzeConversationTaskUnion,
+  AnalyzeConversationOptionalParams,
+  AnalyzeConversationResponse,
+  AnalyzeConversationJobsInput,
+  ConversationAnalysisOptionalParams,
+  ConversationAnalysisResponse
+} from "./generated/models";
+import { conversationAnalysisAzureKeyCredentialPolicy } from "./azureKeyCredentialPolicy";
+import { DEFAULT_COGNITIVE_SCOPE, SDK_VERSION } from "./constants";
+import { isTokenCredential, KeyCredential, TokenCredential } from "@azure/core-auth";
+import { ConversationAnalysisClient as GeneratedClient } from "./generated";
+import { createTracingClient, TracingClient } from "@azure/core-tracing";
+
+/** @internal */
```

Comment:
This class is the exposed one to the customer and shouldn't be marked as internal.
```suggestion
```

Reply:
Could it also be documented properly?